src/components/FilterBar.jsx

```diff
@@ -21,14 +30,52 @@ const amounts = {
   5000: '> 5000',
 };
 
-function FilterBar() {
+const sortingOptions = {
+  deadlineSoon: 'Deadline (Earliest to Latest)',
+  deadlineLatest: 'Deadline (Latest to Earliest)',
+  amountLow: 'Amount (Low to High)',
+  amountHigh: 'Amount (High to Low)',
+};
+
+export default function FilterBar(props) {
+  const { passedFunction } = props;
+
+  function modifySorting(sortOption) {
+    switch (sortOption) {
+      case 'one':
+        return passedFunction('deadline');
+      case 'two':
+        return passedFunction('deadlineReverse');
+      case 'three':
+        return passedFunction('amount.min');
+      case 'four':
+        return passedFunction('amount.max');
+      default:
+        return null;
+    }
+  }
+
+  const classes = useStyles();
   return (
-    <div>
-      <FilterDropDown label="Major" items={majors} />
-      <FilterDropDown label="Grade" items={grades} />
-      <FilterDropDown label="Amount" items={amounts} />
-    </div>
+    <Grid container spacing={2} className={classes.root}>
+      <Grid item>
+        <FilterDropDown label="Major" items={majors} defaultSelect="" />
+        <FilterDropDown label="Grade" items={grades} defaultSelect="" />
+        <FilterDropDown label="Amount" items={amounts} defaultSelect="" />
+      </Grid>
+      <Grid item>
+        <FilterDropDown
+          label="Sorting"
+          items={sortingOptions}
+          defaultSelect="deadlineSoon"
+          removeNone
+          givenFunction={modifySorting}
+        />
```

Comment:
Per the Figma design, none of them are really going to look like our current `FilterDropdown`. You could make a separate `SortDropdown` component that represents the sorting option dropdown.

Reply:
Ok then have that component be used within the `FilterBar` component?

Reply:
Yep. Though in the other PR is fine.